ProjFS.Linux/PrjFSLib.Linux.Managed/NotificationMapping.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace PrjFSLib.Linux
+{
+    public class NotificationMapping
```

Comment:
I would drop this file since it's not being used yet. It's not even clear that it'll be needed in the end (we still haven't decided to add it on Mac, for example)

Reply:
OK ... NotificationMapping.cs is over in the Mac implementation; should I take it out of the Linux one or leave it to stay in sync with the Mac?

Reply:
Ah interesting. I'm actually inclined to take it out of the Mac code, since all we've done there (prematurely it seems) is define the native/managed structs but no functions are using them.

@wilbaker thoughts?

@chrisd8088 no need to block this PR on the decision though, so either way.

Reply:
OK, I might leave it for now then to stay in sync.